### PR TITLE
cpu: mips_pic32*: Fix unused .gcc_except_table section warning

### PR DESCRIPTION
**Pull request by francois-berder**
### Contribution description

When compiling C applications for pic32-clicker and pic32-wifire boards, objcopy always emits a warning about unused section `.gcc_except_table`. This PR removes the parameter to objcopy that causes this warning if this is not an C++ application.

### Testing procedure

When compiling `examples/hello_world` in verbose mode, you should not see `--change-section-lma .gcc_except_table-0x80000000` in objcopy's parameters and the warning should not be printed.
On the other hand, while compiling `examples/riot_and_cpp`, you should see  `--change-section-lma .gcc_except_table-0x80000000` in objcopy's parameters.

### Issues/PRs references

None

### CODE REVIEW

File: cpu/mips_pic32mx/Makefile.include [hunk 1]
```diff
@@ -18,11 +18,14 @@ OFLAGS += \
...
-    --change-section-lma .gcc_except_table-0x80000000 \
     --change-section-lma .jcr-0x80000000 \
     --change-section-lma .ctors-0x80000000 \
     --change-section-lma .dtors-0x80000000 \
     --change-section-lma .rodata-0x80000000 \
     --change-section-lma .data-0x80000000 \
     --change-section-lma .bss-0x80000000 \
     --change-section-lma .startdata-0x80000000 \
+
+ifneq (,$(filter cpp,$(FEATURES_REQUIRED)))
```
Comment (benpicco) [inline cpu/mips_pic32mx/Makefile.include]:
```suggestion
ifneq (,$(filter cpp,$(FEATURES_USED)))
```

File: cpu/mips_pic32mz/Makefile.include [hunk 2]
```diff
@@ -21,11 +21,14 @@ OFLAGS += \
...
-    --change-section-lma .gcc_except_table-0x80000000 \
     --change-section-lma .jcr-0x80000000 \
     --change-section-lma .ctors-0x80000000 \
     --change-section-lma .dtors-0x80000000 \
     --change-section-lma .rodata-0x80000000 \
     --change-section-lma .data-0x80000000 \
     --change-section-lma .bss-0x80000000 \
     --change-section-lma .startdata-0x80000000
+
+ifneq (,$(filter cpp,$(FEATURES_REQUIRED)))
```
Comment (benpicco) [inline cpu/mips_pic32mz/Makefile.include]:
```suggestion
ifneq (,$(filter cpp,$(FEATURES_USED)))
```

File: cpu/mips_pic32mx/Makefile.include (hunk 1 above)
Comment (francois-berder) [reply cpu/mips_pic32mx/Makefile.include]: fixed

File: cpu/mips_pic32mz/Makefile.include (hunk 2 above)
Comment (francois-berder) [reply cpu/mips_pic32mz/Makefile.include]: fixed

**Comment by benpicco** [comment]
You can squash directly.

**Comment by benpicco** [comment]
Seems like you pulled in two unrelated commits 😉

**Comment by francois-berder** [comment]
> Seems like you pulled in two unrelated commits wink

Indeed do not know why it did that. Anyway, I rebased it on top of master.

**Pull request merged**